src/EnergyPlus/BoilerSteam.hh

```diff
@@ -57,6 +57,7 @@
 #include <EnergyPlus/DataGlobals.hh>
 #include <EnergyPlus/EnergyPlus.hh>
 #include <EnergyPlus/PlantComponent.hh>
+#include <EnergyPlus/Plant/DataPlant.hh>
```

Comment:
Is there a reason this couldn't have just pulled in the `Plant/Enums.hh` file instead of the whole `DataPlant.hh` file?

Reply:
This was my error. Will change all of the new DataPlant.hh includes to Enums.hh where possible.

Reply:
OK, sounds great.  Go ahead and do that and apply Clang format and pull develop in, then this _should_ be ready to drop right in.  And thanks!

Reply:
Thank you! :)

Reply:
Running find for "DataPlant.hh" on the GitHub changed files page only revealed five new DataPlant.hh includes and [they have been addressed](https://github.com/NREL/EnergyPlus/pull/9212/commits/96f2d9d21adc72a5c7e4cea46401fa1497dcc8e1). Also merged in develop, and applied Clang format.

Reply:
Awesome!  I'll give CI a little more time to do a few tests and pull it locally as well.  Should go right in if it's all clean.  Thanks!